pycc/rt/integrators.py

```diff
@@ -2,13 +2,94 @@
 integrators.py: various ordinary differential equation solvers for time-domain propagation
 """
 
-__all__ = ['rk2', 'rk3', 'rk38', 'rk4', 'gl6']
+__all__ = ['Euler', 'Midpoint', 'Heun', 'rk2', 'rk3', 'rk38', 'rk4', 'HE', 'Fehlberg', 'BS', 'CK', 'dopri5',
+           'Euler_I', 'Midpoint_I', 'Radau_IA3', 'Radau_IIA3', 'Radau_IA5', 'Radau_IIA5', 'SDIRK5', 'gl4', 'gl6']
 
 import numpy as np
 
+"""
```

Comment:
So are all of these integrators fully implemented?  I see Butcher tableau for some, but there appears to be no explicit implementation for those.

Reply:
The implicit integrators only have the coefficients in the file and not ready to use. They will need more tests for choosing initial guess for the iterations and such. I didn't fully test all of them since they're generally expensive and I only used explicit and adaptive integrators in my calculations. I thought I would keep the Butcher tableau there in case for the future use. Should I remove them since there's no explicit implementation or add a comment to clarify that?

Reply:
Yes, perhaps a comment at the point where the subsequent integrators aren't fully implemented would help.

Reply:
Will do.